tika-parsers/src/main/java/org/apache/tika/parser/envi/EnviHeaderParser.java

```diff
@@ -116,17 +116,86 @@ private void readLines(AutoDetectReader reader, Metadata metadata) throws IOExce
      * Write a line to the XHTMLContentHandler and populate the key, value into the Metadata
      */
     private void writeParagraphAndSetMetadata(String line, Metadata metadata) throws SAXException {
-        if(line.length() < 150) {
+        if(line.length() < 300) {
             String[] keyValue = line.split("=");
```

Comment:
@kristencheung please replace this line
```
String[] keyValue = line.split("=");
```
with this
```
String[] keyValue = line.split("=", 2);
```
This way we do not drop the latter part of the following line entry
```
map info = {Sinusoidal, 1.5000, 1.5000, -10007091.3643, 5559289.2856, 4.6331271653e+02, 4.6331271653e+02, , units=Meters}
```
Thanks